static/progressTracker/steps.js

```diff
@@ -0,0 +1,3 @@
+const steps = ['Draft', 'Book', 'Send invoices', 'Get paid'];
+
+export { steps };
```

Comment:
Doesn't really deserve its own file imo, and isn't a static file 🤔 Just put it in the story itself.